keps/sig-node/2400-node-swap/README.md

```diff
@@ -431,6 +457,7 @@ To avoid exhausting swap on a node, `UnlimitedSwap` was dropped from the API in
 #### Security risk
 
 Enabling swap on a system without encryption poses a security risk, as critical information, such as Kubernetes secrets, may be swapped out to the disk. If an unauthorized individual gains access to the disk, they could potentially obtain these secrets. To mitigate this risk, it is recommended to use encrypted swap. However, handling encrypted swap is not within the scope of kubelet; rather, it is a general OS configuration concern and should be addressed at that level. Nevertheless, it is essential to provide documentation that warns users of this potential issue, ensuring they are aware of the potential security implications and can take appropriate steps to safeguard their system.
+This is now documented and published, e.g. in this blog post: https://kubernetes.io/blog/2023/08/24/swap-linux-beta/.
```

Comment:
```suggestion
The documentation updates are required; there is already a [blog article](https://kubernetes.io/blog/2023/08/24/swap-linux-beta/) that mentions the security implications.
```

1. KEPs describe the “desired state”; we mostly don't document progress inline.
2. Security documentation needs to be done within the guidance itself, such as in https://kubernetes.io/docs/concepts/security/security-checklist/, and not just in blog articles.

Reply:
@iholder101 Is this suggestion applied? If not, why is it marked as resolved?

Reply:
Thanks @bart0sh for pointing that out! I think I've made the changes locally but forgot to push :man_facepalming: 

I've now applied the suggestions (and also the ones mentioned [here](https://github.com/kubernetes/enhancements/pull/4701#discussion_r1757496859)).

Thank you!